src/repository/token.js

```diff
@@ -0,0 +1,9 @@
+const backListTokens = [];
```

Comment:
Another typo, should be `blackListedTokens `. Similar to the function definitions below and where they are called